ui/apps/platform/src/Containers/VulnMgmt/List/Policies/VulnMgmtListPolicies.js

```diff
@@ -1,358 +0,0 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import { gql } from '@apollo/client';
-import pluralize from 'pluralize';
-import { Power, Bell, BellOff, Trash2 } from 'react-feather';
-import { connect } from 'react-redux';
-
-import {
-    defaultHeaderClassName,
-    nonSortableHeaderClassName,
-    defaultColumnClassName,
-} from 'Components/Table';
-import DateTimeField from 'Components/DateTimeField';
-import Dialog from 'Components/Dialog';
-import PolicyDisabledIconText from 'Components/PatternFly/IconText/PolicyDisabledIconText';
-import PolicySeverityIconText from 'Components/PatternFly/IconText/PolicySeverityIconText';
-import PolicyStatusIconText from 'Components/PatternFly/IconText/PolicyStatusIconText';
-import IconWithState from 'Components/IconWithState';
-import PanelButton from 'Components/PanelButton';
-import RowActionButton from 'Components/RowActionButton';
-import TableCountLink from 'Components/workflow/TableCountLink';
-import { formatLifecycleStages } from 'Containers/Policies/policies.utils';
-import WorkflowListPage from 'Containers/Workflow/WorkflowListPage';
-import entityTypes from 'constants/entityTypes';
-import { LIST_PAGE_SIZE } from 'constants/workflowPages.constants';
-import queryService from 'utils/queryService';
-import { workflowListPropTypes, workflowListDefaultProps } from 'constants/entityPageProps';
-import { actions as notificationActions } from 'reducers/notifications';
-import { deletePolicies } from 'services/PoliciesService';
-import removeEntityContextColumns from 'utils/tableUtils';
-import { policySortFields } from 'constants/sortFields';
-
-import { UNSCOPED_POLICY_LIST_FRAGMENT } from 'Containers/VulnMgmt/VulnMgmt.fragments';
```

Comment:
It looks like this is the only reference to the GraphQL fragment.

Reply:
Good catch 👍